doc/man3/SSL_get_peer_certificate.pod

```diff
@@ -9,12 +9,16 @@ SSL_get_peer_certificate - get the X509 certificate of the peer
  #include <openssl/ssl.h>
 
  X509 *SSL_get_peer_certificate(const SSL *ssl);
+ void SSL_peer_certificate_clear(SSL *ssl);
```

Comment:
Just a side note from a noninvolved: IMHO the function name does not follow the naming conventions: I would expect the name  `SSL_clear_peer_certificate`, analogous to `SSL_get_peer_certificate`. See also existing examples like `SSL_clear_chain_certs` and `SSL_clear_options`.

Reply:
Happy to change - there is a SSL_certs_clear which is what I mimicked - https://github.com/openssl/openssl/blob/master/include/openssl/ssl.h#L1744

Reply:
Oh, I see! It looks like OpenSSL itself does not have consistent naming conventions :wink:. Well, in this part of the library  I'm just an uninvolved spectator who happened to read this thread. I'll leave the question of the optimal name to a more qualified reviewer.